scripts/v3.11/php-7.4/php7/APKBUILD

```diff
@@ -356,7 +356,7 @@ _package_ext() {
 		> "$subpkgdir"/etc/$pkgname/conf.d/${elo}_${extname}.ini
 }
 
-sha512sums="75552a89aeb2eb886f9d8310f64f7c2069200b9737ef9fe5a4e3c1766554435156d2f105470c4efc45bddd84afcb8bfa588a437e169199aef89b2b7575869c3a  php-7.4.5.tar.bz2
+sha512sums="92c4a7dd3aa3f55e7501bfa811cdaec94415dda8dd1a55a4a88bee3de2210b1649ba1d5f7f28ceb63247245557402f306a993f432b535f4c3adb1204c5e51184  php-7.4.9.tar.bz2"
```

Comment:
It looks like you've accidentally left a trailing double quote on the end of this line for the 7.4 build on Alpine 3.9, 3.10, and 3.11

Reply:
that's not an actual problem since this line is generated by `abuild` command, and updated after downloading the sources to generate the checksum